app/Plugin/LdapProvisioner/View/CoLdapProvisionerTargets/fields.inc

```diff
@@ -293,16 +300,18 @@
             $ocopts = array();
             $ocopts['label'] = _txt('pl.ldapprovisioner.oc.enable', array($oc));
             $ocopts['onClick'] = 'toggle_div("oc_' . $oc . '_div")';
-
+            $ocopts['checked'] = false;
             if($supportedAttributes[$oc]['objectclass']['required']) {
               $ocopts['label'] .= " (" . _txt('fd.required') . ")";
               $ocopts['checked'] = true;
               $ocopts['disabled'] = true;
             }
-
+            if(!empty($co_ldap_provisioner_targets[0]['CoLdapProvisionerTarget']['oc_' . strtolower($oc)])){
+              $ocopts['checked'] = true;
+            }
             print $this->Form->input('oc_' . strtolower($oc), $ocopts) . "<br />";
 
-            print '<div id="oc_' . $oc . '_div">';
+            print '<div id="oc_' . $oc . '_div" class="nested">';
```

Comment:
class nested is missing from co-base.css? Have you tested how ui will behave if you import that rule?